serverless.yml

```diff
@@ -0,0 +1,61 @@
+service: dynamodb-backup
+frameworkVersion: "=1.26.0"
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+  region: ap-southeast-2
+
+  iamRoleStatements:
+    - Effect: "Allow"
+      Action:
+        - "logs:CreateLogGroup"
+        - "logs:CreateLogStream"
+        - "logs:PutLogEvents"
+      Resource: "arn:aws:logs:*:*:*"
+    - Effect: "Allow"
+      Action:
+        - "s3:ListBucket"
+      Resource: "<YOUR BACKUP BUCKET NAME>"
```

Comment:
If the environment variables were global (instead of per-function), we could retrieve this without users having to replace them manually